docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -1,11 +1,45 @@
 # Converting a PyTorch Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch}
 
-The PyTorch framework is supported through export to the ONNX format. In order to optimize and deploy a model that was trained with it:
+The PyTorch framework is supported through export to the ONNX format. Model Optimizer Python API allows the conversion of PyTorch models using the `convert_model()` method, which internally converts a model to ONNX.
+
+## Converting a PyTorch model from memory with Python API (Experimental Functionality)
+
+To convert a PyTorch model using `convert_model()`, provide the `input_shape` or `example_input`.
```

Comment:
Add more words about when we should use `input_shape` or `example_input`. Describe how input_shape is really used, especially in case of dynamic dimensions.

Reply:
Added details. Please check.